cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -134,7 +133,13 @@ runNode nodeCli@NodeCLIArguments{..} loggingLayer cc = do
 
       SimpleNode topology myNodeAddress protocol viewMode traceOptions -> do
         let trace'      = appendName (pack $ show $ node topology) tr
+        let tracer      = contramap pack $ toLogObject trace'
 
+        traceWith tracer $ "tracing verbosity = " ++
+                             (case (traceVerbosity traceOptions) of
```

Comment:
I think you can remove both pairs of parentheses.

Reply:
cleaned up